working-with-fhir-igs/au-core/README.md

```diff
@@ -0,0 +1,24 @@
+# FHIR AU Core Sample
+
+This repository contains a sample usage of Ballerina FHIR AU Core package to work with FHIR resources.
+
+## Prerequisites
+
+Before running this sample, make sure you have the following:
+
+- Download and install [Ballerina Swan Lake](https://ballerina.io/downloads/) 2201.8.5 or above.
+
+## Getting Started
+
+To get started with this sample, follow these steps:
+
+1. Clone this repository to your local machine.
+2. Run the project using the following command:
```

Comment:
Do we need a step asking to navigate to the relevant project?

Reply:
+1. added change in f68c3145f92c07b83a485f31c7d0b451dc782aba